test_unstructured/partition/pdf_image/test_pdf.py

```diff
@@ -998,11 +999,18 @@ def test_check_annotations_within_element(threshold, expected):
 
 @pytest.fixture(scope="session")
 def chipper_results():
-    elements = pdf.partition_pdf(
-        "example-docs/layout-parser-paper-fast.pdf",
-        strategy="hi_res",
-        model_name="chipper",
-    )
+    parent = layout.LayoutElement.from_coords(0, 1, 2, 4, text="")
+    chipper_output = [
+        parent,
+        layout.LayoutElement.from_coords(5, 6, 7, 8, text="child", parent=parent),
+    ]
+
+    with mock.patch.object(UnstructuredChipperModel, "predict", return_value=chipper_output):
+        elements = pdf.partition_pdf(
+            "example-docs/layout-parser-paper-fast.pdf",
+            strategy="hi_res",
+            model_name="chipper",
+        )
```

Comment:
there ought to be a real test for chipper somewhere in CI though.
can the test be against a much smaller file?

Reply:
We do test the chipper model in inference.

Reply:
that's not the same. maybe the real inference belongs in a separate CI job?